sdk/core/azure-core/CMakeLists.txt

```diff
@@ -100,18 +103,21 @@ target_include_directories(
   azure-core
     PUBLIC
       $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
-      $<INSTALL_INTERFACE:include/az_core>
+      $<INSTALL_INTERFACE:include>
 )
 
 # make sure that users can consume the project as a library.
-add_library(Azure::Core ALIAS azure-core)
+add_library(Azure::azure-core ALIAS azure-core)
```

Comment:
what happen if we leave it as Azure::Core ?  
Looks weird to repeat azure twice

Reply:
It tells me that Azure::Core is an alias and can't be exported. If I simply use it in this specific place, the target library vcpkg produces is still Aaure::azure-core. So then when you build another lib as vcpackage, and we find_package() and then try to reference it as Azure::Core, it fails to find it.